Bottom: cd221061d70b2a9a26468a9c04f5bcd0752d1645
Top:    1cc3e3245a7bab79c626d37c54a44f7d397f9fad
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 08:46:53 -0500

Install buildbot plugin dependencies


---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 3e48e26e..e888342f 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -14,7 +14,11 @@ RUN apk --no-cache add \
 
 # install pip dependencies
 RUN pip --no-cache-dir install \
-    'attrs'
+    'attrs' \
+    'buildbot-console-view' \
+    'buildbot-grid-view' \
+    'buildbot-waterfall-view' \
+    'buildbot-www'
 
 # Create buildbot user
 ARG BUILDBOT_UID=1000
